src/main/java/frc/robot/subsystems/scoring/ScoringSubsystem.java

```diff
@@ -37,6 +38,8 @@ public class ScoringSubsystem extends SubsystemBase {
     private final InterpolateDouble shooterInterpolated;
     private final InterpolateDouble aimerInterpolated;
 
+    private Translation2d fieldToSpeaker = FieldConstants.fieldToRedSpeaker;
```

Comment:
Maybe we should make a part of RobotContainer supply the applicable field constants to this and the drivetrain so we don't need to duplicate figuring out which speaker we want.